cmake/USDPluginTools.cmake

```diff
@@ -467,6 +490,23 @@ function(_usd_cpp_library NAME)
             PREFIX "${LIBRARY_FILE_PREFIX}"
     )
 
+    # Add MSVC definitions
+    if (MSVC)
+        # Set the language standard
+        set_property(TARGET ${NAME} 
+            PROPERTY 
+                CXX_STANDARD 14)
+
+        # Add required compile definitions for MSVC.
+        # Depending on which parts of USD the project uses, additional definitions for windows may need
+        # to be added. A explicit list of MSVC definitions USD builds with can be found in the USD source at:
+        #   cmake/defaults/CXXDefaults.cmake
+        #   cmake/defaults/msvcdefaults.cmake
+        add_compile_definitions(
+            NOMINMAX;
+        )
```

Comment:
Cheers for adding the comment / explanation - it'll definitely help future maintainers.

Could you move this MSVC-specific definition addition into the `_usd_target_properties` implementation, and could you write the logic to pass these additional definitions into the `target_compile_definitions` call?  Thanks!